diff --git a/package/amlogic/ipc/mbp/mbi/mbi.mk b/package/amlogic/ipc/mbp/mbi/mbi.mk
index c9f3daeeb9..83c53743d7 100644
--- a/package/amlogic/ipc/mbp/mbi/mbi.mk
+++ b/package/amlogic/ipc/mbp/mbi/mbi.mk
@@ -29,7 +29,7 @@ MBI_GIT_VERSION=$(shell \
 			else \
 			   echo "unknown"; \
 			fi)
-MBI_CFLAGS = \
+MBI_CFLAGS += \
     -I$(LINUX_OSAL_DIR)/include \
     -I$(MBD_BASE_DIR)/include \
     -I$(MBD_BASE_DIR)/vbp/include \
@@ -93,7 +93,7 @@ define MBI_BUILD_CMDS
 	REVISION=$(MBI_GIT_VERSION) $(TARGET_CONFIGURE_OPTS) $(TARGET_MAKE_ENV) CFLAGS="$(MBI_CFLAGS)" $(MAKE) -C $(@D)/mbi_base
 	REVISION=$(MBI_GIT_VERSION) $(TARGET_CONFIGURE_OPTS) $(TARGET_MAKE_ENV) CFLAGS="$(MBI_CFLAGS)" $(MAKE) -C $(@D)/venc
 	REVISION=$(MBI_GIT_VERSION) $(TARGET_CONFIGURE_OPTS) $(TARGET_MAKE_ENV) CFLAGS="$(MBI_CFLAGS)" $(MAKE) -C $(@D)/cve
-	REVISION=$(MBI_GIT_VERSION) $(TARGET_CONFIGURE_OPTS) $(TARGET_MAKE_ENV) CFLAGS="$(MBI_CFLAGS)" $(MAKE) -C $(@D)/adla
+	#REVISION=$(MBI_GIT_VERSION) $(TARGET_CONFIGURE_OPTS) $(TARGET_MAKE_ENV) CFLAGS="$(MBI_CFLAGS)" $(MAKE) -C $(@D)/adla
 	REVISION=$(MBI_GIT_VERSION) $(TARGET_CONFIGURE_OPTS) $(TARGET_MAKE_ENV) CFLAGS="$(MBI_CFLAGS)" $(MAKE) -C $(@D)/dewarp
 	REVISION=$(MBI_GIT_VERSION) $(TARGET_CONFIGURE_OPTS) $(TARGET_MAKE_ENV) CFLAGS="$(MBI_CFLAGS)" $(MAKE) -C $(@D)/isp
 	REVISION=$(MBI_GIT_VERSION) $(TARGET_CONFIGURE_OPTS) $(TARGET_MAKE_ENV) CFLAGS="$(MBI_CFLAGS)" $(MAKE) -C $(@D)/ppu
@@ -109,7 +109,7 @@ define MBI_INSTALL_STAGING_CMDS
 	$(INSTALL) -D -m 644 $(@D)/cve/include/* $(STAGING_DIR)/usr/include/
 	$(INSTALL) -D -m 644 $(@D)/cve/cva/lib32/*.so $(STAGING_DIR)/usr/lib/
 	$(INSTALL) -D -m 644 $(@D)/cve/cva/include/* $(STAGING_DIR)/usr/include/
-	$(INSTALL) -D -m 644 $(@D)/adla/libmbi_adla.so $(STAGING_DIR)/usr/lib/
+	#$(INSTALL) -D -m 644 $(@D)/adla/libmbi_adla.so $(STAGING_DIR)/usr/lib/
 	$(INSTALL) -D -m 644 $(@D)/adla/include/* $(STAGING_DIR)/usr/include/
 	$(INSTALL) -D -m 644 $(@D)/dewarp/libmbi_dewarp.so $(STAGING_DIR)/usr/lib/
 	$(INSTALL) -D -m 644 $(@D)/dewarp/include/* $(STAGING_DIR)/usr/include/
diff --git a/package/amlogic/ipc/mbp/sample/sample.mk b/package/amlogic/ipc/mbp/sample/sample.mk
index 2819e6e83e..5435162071 100644
--- a/package/amlogic/ipc/mbp/sample/sample.mk
+++ b/package/amlogic/ipc/mbp/sample/sample.mk
@@ -46,7 +46,7 @@ define SAMPLE_LACK_WARNING
 		@printf '\033[1;33;40m[WARNING]  %b\033[0m\n' "SAMPLE: LACK of prebuilt release filelist!"
 endef
 
-SAMPLE_CFLAGS = \
+SAMPLE_CFLAGS += \
     -I$(SAMPLE_DIR)\
     -I$(SAMPLE_DIR)/pipeline_demo\
     -I$(LINUX_OSAL_DIR)/include \
