diff --git a/Makefile b/Makefile
index a18bbec..9efa366 100644
--- a/Makefile
+++ b/Makefile
@@ -1,21 +1,21 @@
-SAMPLE_TARGETS += vbp
-SAMPLE_TARGETS += pmz
-SAMPLE_TARGETS += log
-SAMPLE_TARGETS += sys
-SAMPLE_TARGETS += cve
-SAMPLE_TARGETS += cva
-SAMPLE_TARGETS += ge2d
+#SAMPLE_TARGETS += vbp
+#SAMPLE_TARGETS += pmz
+#SAMPLE_TARGETS += log
+#SAMPLE_TARGETS += sys
+#SAMPLE_TARGETS += cve
+#SAMPLE_TARGETS += cva
+#SAMPLE_TARGETS += ge2d
 SAMPLE_TARGETS += venc
 SAMPLE_TARGETS += venc/cli
-SAMPLE_TARGETS += region
-SAMPLE_TARGETS += adla
-SAMPLE_TARGETS += dewarp
-SAMPLE_TARGETS += isp
+#SAMPLE_TARGETS += region
+#SAMPLE_TARGETS += adla
+#SAMPLE_TARGETS += dewarp
+#SAMPLE_TARGETS += isp
 SAMPLE_TARGETS += pipeline_demo
-SAMPLE_TARGETS += vpu
-SAMPLE_TARGETS += audio
-SAMPLE_TARGETS += audio_codec
-SAMPLE_TARGETS += audio_codec/asp
+#SAMPLE_TARGETS += vpu
+#SAMPLE_TARGETS += audio
+#SAMPLE_TARGETS += audio_codec
+#SAMPLE_TARGETS += audio_codec/asp
 
 all:
 	for f in $(SAMPLE_TARGETS); do if [ -d "$$f" ]; then ($(MAKE) -C $$f) || exit 1; fi; done
